www/zmq/util.go

```diff
@@ -0,0 +1,30 @@
+package zmq
+
+import (
+	"encoding/binary"
+
+	"github.com/pactus-project/pactus/crypto"
+)
+
+func makeTopicMsg(parts ...any) []byte {
```

Comment:
I feel we can add it to the base and then add topic and seq at the beginning and end of all topic messages.